doc/classes/ImmediateGeometry.xml

```diff
@@ -75,7 +76,7 @@
 			<argument index="0" name="normal" type="Vector3">
 			</argument>
 			<description>
-				Set the normal that the next vertex will use to be drawn.
+				Sets the normal that the next vertex will use to be drawn.
```

Comment:
Ref writing guidelines: "Don’t repeat the method’s name. It already states it sets":

    The next vertex's normal.